packages/app/cypress/e2e/debug.cy.ts

```diff
@@ -147,5 +162,12 @@ describe('App - Debug Page', () => {
     cy.findByTestId('test-row').contains('InfoPanel')
     cy.findByTestId('test-row').contains('renders')
     cy.findByTestId('run-failures').should('exist').should('have.attr', 'href', '#/specs/runner?file=src/components/InfoPanel/InfoPanel.cy.ts&mode=debug')
+
+    cy.findByTestId('open-in-ide').click().then(() => {
```

Comment:
Hmm do you need `then`? Shouldn't `cy.wait` be enough?

Reply:
This was more of a style/grouping, I wanted to keep the assertions logically grouped so I grouped them within the `.then`. Not needed and can revert if it reads better without it

Reply:
Removed in [31bdc36](https://github.com/cypress-io/cypress/pull/25691/commits/31bdc36828db6a757c9ac651c42eb8366f3f6a34)